### ISSUE TITLE
Quest: "Multiple Devices Can't Access This App At The Same Time"

### ISSUE THREAD

**Issue by ipeakermit** [icosa-foundation/open-brush#785]
We are setting up to use OpenBrush in passthrough mode for up to 20 people simultaneously using a fleet of Quest 3s, which for ease of management are all covered to use a common primary (developer) account. This is for use in a common physical space for workshops / focus groups / tutorials etc.

However, users on the second and all subsequent devices running OpenBrush see the error "Multiple Devices Can't Access This App At The Same Time". This appears to be a known issue when using any/all apps installed via the Meta Store, even free ones.
 
Having seen the error on the free Meta Store version of OpenBrush, we tried instead sideloading of a recent OpenBrush APK. However it appears that the same issue occurs when using the sideloaded OpenBrush.

Is this a known issue? Are there any workarounds other than creating multiple Meta accounts, one for each separate device?

**Comment by mikeage** [comment]
Ugh. Yeah.

Sorry, that's basically all I have.

It's a Meta limitation, and we don't seem to have any way to opt out. It's a free app, and we want people using it, but as long as you're online, it does this check.

As you saw, it identifies apps based on their signature and package name, which is the same whether you install (sideload) from an APK directly or via the store.

You might be able to take one of the APKs built from a PR, as those have different names. (each PR has a unique name). I don't think, but I may be wrong, that Meta enforces concurrency on these type of apps.

The two downsides are that you don't get updates, and you'll have to launch it from "Unknown Sources" without a nice icon. Depending on your audience, I don't know if that's a blocker.

Wish I had something better to offer.

**Comment by ipeakermit** [comment]
Thanks! I appreciate the quick guidance; it will help to know what to try and what NOT to try.

As a fallback, we seem to be having success going with new individual (non-developer) Meta accounts.

**Comment by ipeakermit** [comment]
This was resolved by modifying the package name in the APK I believe. Will try to get clarity on that.

**Comment by mikeskydev** [comment]
Yeah that makes sense, it no longer believes it's an official app at that point and you can do what you want
